src/winstanley/wdl.bnf

```diff
@@ -32,7 +32,11 @@ workflow_block ::= WORKFLOW WORKFLOW_IDENTIFIER_DECL LBRACE wf_body_element* RBR
 wf_body_element ::= call_block | input_block | declaration | while_loop | if_stmt | scatter_block | wf_outputs
 
 input_block ::= INPUT LBRACE input_declaration* RBRACE
-input_declaration ::= type_e IDENTIFIER {mixin="winstanley.psi.impl.WdlNamedElementImpl" implements="winstanley.psi.WdlNamedElement" methods=[getName getNameIdentifier setName]}
+input_declaration ::= type_e IDENTIFIER (EQUAL expression)? {
```

Comment:
This actually might now be the same as the `declaration` below. I **think** we probably should consolidate.

Reply:
Excellent point, I consolidated and everything works, so let's go with that